CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Accessibility - Make H1 readable by screen readers
 - Public: Accessibility - Make buttons/links readable by screen readers, allow tabbing to them
 - Public: Accessibility - Announce page transition when screen changes
+- Public: Accessibility - Make capture previews readable by screen readers
```

Comment:
Accessibility is actually UI

Reply:
@rfreitas Should we have a separate PR, or fix that when release happens?

Reply:
@rfreitas Fixed (previous entries) with a PR. @stefaniacardenas please if you could sync the branch and change your entry.